h/static/scripts/tests/controllers/search-bar-controller-test.js

```diff
@@ -250,6 +397,15 @@ describe('SearchBarController', function () {
       return ctrl.refs.searchBarLozenges.querySelectorAll('.js-lozenge');
     }
 
+    /**
+     * return the array of all of lozenges values
+     */
+    const getLozengeValues = function (){
+      return Array.from(ctrl.refs.searchBarLozenges.querySelectorAll('.lozenge')).map((el)=>{
+        return el.querySelector('.lozenge__content').innerText;
```

Comment:
Doesn't prevent merge but I'll note that [`innerText` is a non-standard property](https://developer.mozilla.org/en-US/docs/Web/API/Node/innerText).